src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml

```diff
@@ -81,7 +94,39 @@ tests:
           [1657021528.886209][7089:7094] CHIP:TOO:     [23]: 65531
           [1657021528.886234][7089:7094] CHIP:TOO:     [24]: 65532
           [1657021528.886258][7089:7094] CHIP:TOO:     [25]: 65533
-      disabled: true
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains:
+                  [
+                      0,
+                      1,
+                      2,
+                      3,
+                      4,
+                      5,
+                      6,
+                      7,
+                      8,
+                      9,
+                      10,
+                      11,
+                      12,
+                      13,
+                      14,
+                      15,
+                      16,
+                      17,
+                      18,
```

Comment:
Attributes 11-18 are all optional, so should probably not be tested for here unless the right PICS are set.  Just take those out of the list?

Reply:
Removed optional.